doc/user/layouts/shortcodes/mysql-direct/ingesting-data/use-aws-privatelink.html

```diff
@@ -2,127 +2,136 @@
 client connected to Materialize, use the [`CREATE CONNECTION`](/sql/create-connection/#aws-privatelink)
 command to create an AWS PrivateLink connection:
 
-    ↕️ **In-region connections**
+↕️ **In-region connections**
```

Comment:
Oh ... this isn't rendering properly.  You seem to have accidentally removed the whitespaces

https://preview.materialize.com/materialize/32414/ingest-data/mysql/amazon-aurora/#2-start-ingesting-data